src/test/java/FilterServiceTest.java

```diff
@@ -0,0 +1,66 @@
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import ru.jru.quest.karaganov.service.FilterService;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.mock;
+
+@ExtendWith(MockitoExtension.class)
+public class FilterServiceTest {
+
+
+    @Mock
+    static HttpServletRequest request = mock(HttpServletRequest.class);
```

Comment:
Why do we need both static method `mock` and annotation?

Reply:
@Mock deleted